misaligned.c

```diff
@@ -1,21 +1,98 @@
 #include <stdio.h>
 #include <assert.h>
+#include <string.h>
+
+#define MAX_COLOR_PAIR 25
+#define MAX_COLOR_LEN 15
+
+typedef struct {
+    char majorColor[MAX_COLOR_LEN];
+    char minorColor[MAX_COLOR_LEN];
+}ColorPair;
+
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+ColorPair colorPairs[MAX_COLOR_PAIR];
+
+int getColorMapList() {
 
-int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            strcpy(colorPairs[(i*5)+j].majorColor, majorColor[i]);
+            strcpy(colorPairs[(i*5)+j].minorColor, minorColor[j]);
         }
     }
     return i * j;
 }
 
+void printColorMap() {
+    int size, i;
+    size = getColorMapList();
+
+    for (i = 0; i < size; i++) {
+        printf("%d | %s | %s\n", i, colorPairs[i].majorColor, colorPairs[i].minorColor);

```

Comment:
zero based color pair is printed... is this expected from a user point of view? or the color pair number needs to start from 1?
Also if that is the case, is there a test case to confirm that expectation?